app/src/main/res/values-it/strings.xml

```diff
@@ -206,16 +239,22 @@
     <string name="intro_label">Introduzione</string>
     <string name="welcome_to_vinyl_music_player">"Benvenuto in Vinyl Music Player, un music player bellissimo e leggero per Android. "</string>
     <string name="open_playing_queue_instruction">Fai uno swipe sulla card nella schermata In riproduzione per mostrare l\'intera coda di riproduzione.</string>
-    <string name="rearrange_playing_queue_instruction">Riorganizza la coda di riproduzione trascinando una canzone dal suo numero di traccia.</string>
+    <string name="rearrange_playing_queue_instruction">Riorganizza la coda di riproduzione trascinando un brano dal suo numero di traccia.</string>
     <string name="library">Libreria</string>
     <string name="folders">Cartelle</string>
     <string name="saved_playlist_to">Playlist salvata in %s.</string>
     <string name="failed_to_save_playlist">Salvataggio playlist fallito (%s).</string>
+    <string name="saved_x_playlists_to_x">%1$d playlist salvate in %2$s.</string>
+    <string name="saved_x_playlists_to_x_failed_to_save_x">Salvate %1$d playlist in %2$s, mentre è fallito il salvataggio di %3$d.</string>	
+    <string name="some_files_are_not_listed_in_the_media_store">Alcuni file non sono elencati nel media store.</string>
     <string name="not_listed_in_media_store"><![CDATA[<b>%s</b> non è elencato nel media store.]]></string>
     <string name="nothing_to_scan">Nulla da scansionare.</string>
     <string name="scanned_files">Scansione effettuata per %1$d su %2$d file.</string>
     <string name="could_not_scan_files">Non è stato possibile effettuare la scansione di %d file.</string>
-    <string name="listing_files">Creazione elenco dei file</string>
+    <string name="listing_files">Creazione elenco dei brani...</string>
+    <string name="scanning_x_songs_in_progress">Aggiornamento di %1$d brani…</string>
+    <string name="scanning_x_songs_finished">Aggiornati %1$d brani</string>
+    <string name="listing_files">Lettura dei file</string>					
```

Comment:
This is creating a build error since `listing_files` is duplicated

Reply:
Sorry. Do you want me to send a new PR?

Reply:
Yes please

Reply:
Done #477